sbi/neural_nets/classifier.py

```diff
@@ -0,0 +1,170 @@
+# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
+# under the Affero General Public License v3, see <https://www.gnu.org/licenses/>.
+
+from __future__ import annotations
+
+import torch
+from pyknos.nflows.nn import nets
+from torch import Tensor, nn, relu
+
+from sbi.utils.sbiutils import standardizing_net
+
+
+def build_input_layer(
+    batch_x: Tensor = None,
+    batch_y: Tensor = None,
+    z_score_x: bool = True,
+    z_score_y: bool = True,
+) -> nn.Module:
+    """Builds input layer for classifers that optionally z-scores
```

Comment:
Please put a sentence on what x is and what y is.

Or even call them theta and x